src/cli/cli.ts

```diff
@@ -91,7 +93,7 @@ import {
   const resolutionProcess: Promise<boolean>[] = [];
   // Execute resolution for each currency
   if (options.currencies) {
-    options.currencies.forEach(async (currency:string) => {
```

Comment:
I don't actually know if this async were doing anything here. TryInfo is running an async function but it doesn't seem we are waiting for it